plugins/plugin-svelte/package.json

```diff
@@ -19,10 +19,6 @@
   "dependencies": {
     "rollup-plugin-svelte": "^7.0.0",
     "svelte-hmr": "^0.13.2",
-    "svelte-preprocess": "^4.6.0"
-  },
-  "devDependencies": {
-    "node-sass": "^5.0.0",
```

Comment:
This devDep was single-handedly blocking Node 16 builds ([issue](https://github.com/sass/node-sass/issues/3077)). But it’s a devDep, so it’s fairly safe to remove.